apps/web/src/pages/Pools/PoolLayout.tsx

```diff
@@ -104,9 +123,15 @@ const PoolHeader = ({ poolQuery }: PoolHeaderProps) => {
               }
         }
         contentRight={
-          <Chip color={advertisementBadge.color} data-h2-flex-shrink="base(0)">
-            {intl.formatMessage(advertisementBadge.label)}
-          </Chip>
+          (currentPage?.link.url.includes("edit") ||
+            currentPage?.link.url.includes("plan")) && (
+            <Chip
+              color={advertisementBadge.color}
+              data-h2-flex-shrink="base(0)"
+            >
+              {intl.formatMessage(advertisementBadge.label)}
+            </Chip>
+          )
```

Comment:
We need to update this to use the validation for the assessment plan builder badge when on said page. 

https://github.com/GCTC-NTGC/gc-digital-talent/blob/9637-advertisement-assessment-pages/apps/web/src/utils/poolUtils.tsx#L420-L424

```tsx
export const getPoolCompletenessBadge = (completeness: PoolCompleteness) => {
  const badgeInfo = poolCompletenessMap.get(completeness);


  return badgeInfo ?? defaultCompleteness;
};
```

Reply:
See: 05e481b764b3add25120500807cc5aef8c8ab8c4

Reply:
I'm still seeing an issue. However, it's not on the assessment plan builder page, that seems to working correctly, so you got that right. We just need to transfer what you did to ViewPoolPage.tsx as well, which might just be adding the missing fields. However, that's getting handled on [Vachan's PR](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10237) 👍 so this should be good to go.

https://github.com/GCTC-NTGC/gc-digital-talent/assets/22059495/88bc01dd-3ee7-4b1e-affd-c83b4005b124

Reply:
@yonikid15 maybe wait until #10237 is merged into `main` and then merge `main` into this branch to verify it is actually fixed?

Reply:
Ya we can wait 👍

Reply:
@yonikid15 merged and ready for re-review: edaa39a4fc9bf5c457418e664dfe0bf76808919b.